app/models/petition.rb

```diff
@@ -72,6 +71,9 @@ class Petition < ActiveRecord::Base
   facet :tagged_in_moderation,         -> { tagged_in_moderation.by_most_recent_moderation_threshold_reached }
   facet :untagged_in_moderation,       -> { untagged_in_moderation.by_most_recent_moderation_threshold_reached }
 
+  facet :collecting_signatures, -> { not_archived.open_state.by_most_popular }
```

Comment:
This order needs changing - `by_most_recently_published`, i.e. `reorder(open_at: :desc)`